README.md

```diff
@@ -1 +1,29 @@
-# minterop-data
+# Minterop Data
+
+## Postgres and Hasura GraphQL/REST API configuration for Mintbase Indexers
+
+Configurations here manage both **postgres database column migrations** and **hasura configuration** in a single script that can be run in one of three environments **sandbox (mainnet NEAR data)**, **testnet (testnet NEAR data)**, and **mainnet**.
+
+Running `scripts/redeploy.sh [env]` can be done locally but will require a {environment_name}.env file will need to be present. Ask if you need help with the contents of this file.
+
+**Note, local migrations should only ever be done run sandbox because they lack the inconsitency checks in CICD**
```

Comment:
Maybe we should include the `hasura metadata ic list` in the script, so we at least get a warning when we have problems on mainnet/testnet.

Reply:
I can do that on next iteration if you leave me a TODO comment in there :)